canopy/canopyjs/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "compilerOptions": {
     "lib": ["es2015", "dom"],
     "esModuleInterop": true,
-    "declaration": true
+    "declaration": true,
+    "resolveJsonModule": true,
+    "allowJs": true
```

Comment:
does this mean we lose types for the protobuf library?

Reply:
No since the types file is generated.